INSTALL.md

```diff
@@ -59,6 +63,12 @@ To produce an executable installer on Windows, the following are required:
 1. [Node.JS and NPM](<https://www.npmjs.com/get-npm>)
     1.  Install version 10.15.0 LTS
     
+##### Code Signing (optional)
+
+For code signing to work, you will need to set the `HF_PFX_FILE` and `HF_PFX_PASSPHRASE` environment variables globally.
```

Comment:
And at runtime, per previous code-signing comment change, above?

Reply:
I don't believe runtime is necessary, runtime alone will not work. So if I put comments above, I should undo that. ;)

Reply:
Oh, sorry, so I read it and that is correct... what I meant is the code signing certificate needs to be set when running CMake and when packaging as well. :)

Reply:
Add something along the lines of, "as well as setting when running CMake"? And refer to how to set when running CMake?